install_config/configuring_sdn.adoc

```diff
@@ -159,12 +159,14 @@ but not to *172.28.0.0/14*, because even though the original range is entirely
 inside the new range, the original range must be at the start of the CIDR. See
 xref:expanding-the-service-network[Expanding the service network] for more
 information.
+====
 
+[[config-changing-vxlan-port-for-cluster-network]]
+== Changing the VXLAN PORT for the cluster network
 
-.Changing the VXLAN PORT for the cluster network
-
-A cluster administrator can change the VXLAN port the system uses by changing the vxlanPort variable in the
-master configuration file.
+As a cluster administrator, you can change the VXLAN port the system uses by
+changing the `vxlanPort` variable in the master configuration file at
```

Comment:
I'd s/file at/file, located in the <dir> directory
You should probably at least s/file at/file, located in